src/pages/console/uibuilder/collections.mdx

```diff
@@ -20,18 +37,49 @@ Amplify Studio offers a visual way to bind collections to items in your data mod
 
 ### Set filters
 
-Filters control what records are displayed in a collection.
+Filters control what records are displayed in a collection. You can add multiple filters to a single collection.
 
 ![Screenshot showing setting filters](/images/console/ui-collections-set-filters.png)
 
 ### Set sort conditions
 
 Sort conditions control the direction records are displayed in a collection.
 
+<Callout warning>
+
+Collections bound to an API without DataStore cannot be sorted. [Learn more about DataStore and Conflict Resolution](/console/data/data-model/#datastore-and-graphql)
```

Comment:
```suggestion
Collections bound to an GraphQL API without DataStore cannot be sorted. [Learn more about DataStore and Conflict Resolution](/console/data/data-model/#datastore-and-graphql)
```
customer have expressed confusion about a API, as Amplify supports both REST and GraphQL. switching between docs can get confusing